CMakeLists.txt

```diff
@@ -1,44 +1,35 @@
-cmake_minimum_required(VERSION 3.1)
+cmake_minimum_required(VERSION 3.18)
+include(FindPkgConfig)
 
 project(kq-fork)
 
-set (CMAKE_MODULE_PATH "${CMAKE_MODULE_PATH};${CMAKE_CURRENT_SOURCE_DIR};${CMAKE_CURRENT_SOURCE_DIR}/maps")
+set (CMAKE_MODULE_PATH "${CMAKE_MODULE_PATH}"
+  "${CMAKE_CURRENT_SOURCE_DIR}")
 
-find_package(Allegro REQUIRED)
+find_package(SDL2 CONFIG REQUIRED)
```

Comment:
Where can I find the SDL.cmake file that would go with this? Or, how do I generate whichever file it's needing?

Reply:
What platform are you using? The relevant package is part of SDL and should have been installed by your system's package manager.

Reply:
Windows 10. The sdl install instructions only say to extract the files somewhere, but nothing concrete.

I miss Linux.